src/store/index.js

```diff
@@ -0,0 +1,8 @@
+import { createStore, applyMiddleware } from 'redux';
+import { createLogger } from 'redux-logger';
+import allReducers from '../reducers';
+
+const logger = createLogger();
+
+export const store = createStore(allReducers, applyMiddleware(logger));
```

Comment:
Why have you used logger instead of thunk in middleware??

Reply:
I have fixed this. I'm using `thunk` now